src/pages/storage/EditStoragePool.tsx

```diff
@@ -27,6 +30,7 @@ const EditStoragePool: FC<Props> = ({ pool }) => {
   const notify = useNotify();
   const queryClient = useQueryClient();
   const { project } = useParams<{ project: string }>();
+  const [section, setSection] = useState(MAIN_CONFIGURATION);
```

Comment:
Same as above:
Probably better to move this state into `StoragePoolForm` as it is not needed here.